test/fakeintake/app/main.go

```diff
@@ -19,7 +19,14 @@ func main() {
 	signal.Notify(sigs, os.Interrupt, syscall.SIGTERM)
 
 	fmt.Println("⌛️ Starting fake intake")
-	fi := fakeintake.NewServer(8080)
+	ready := make(chan bool, 1)
```

Comment:
I am not sure the `main` should block on `ready`. If there's any mistake in `Start()`, it will hang in there forever.

Reply:
You are right, I should at least add a timeout tick

Reply:
Added it